providers/nodeReleasesProvider.tsx

```diff
@@ -0,0 +1,16 @@
+import { createContext } from 'react';
+import { useFetchNodeReleases } from '../hooks/useFetchNodeReleases';
+import type { FC, PropsWithChildren } from 'react';
+import type { NodeRelease } from '../types';
+
+export const NodeReleasesContext = createContext<NodeRelease[]>([]);
+
+export const NodeReleasesProvider: FC<PropsWithChildren> = ({ children }) => {
+  const releases = useFetchNodeReleases();
+
+  return (
+    <NodeReleasesContext.Provider value={releases}>
```

Comment:
We should wrap children with the SWR Provider with the fallback for Node Release Data so that we have Server-Side SWR :)

Reply:
I assume the usage is something like this?
```
<NodeReleasesProvider fallback={nodeReleasesServerSide}>
  ...
</NodeReleasesProvider>
```

Reply:
My bad... lol
https://swr.vercel.app/docs/with-nextjs#pre-rendering-with-default-data

Reply:
@ovflowd I saw you resolved this? Or we are going to do it later?

Reply:
Oh I thought you did it already. But it would be:

```tsx
<NodeReleasesContext.Provider value={releases}>
  <SWRConfig value={{ fallback }}>
    {children}
  </SWRConfig>
</NodeReleasesContext.Provider>
```

fallback = the fallback with the server-side data for the releases. Since I don't have a clear answer on how we can do this (like fetch the data only once on a `getStaticProps` 

I think we should create a separate issue and check this later, not in this PR.

Since we removed the legacyMain stuff it should be stable now.

Can you create an issue?